WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/RefreshableInfiniteScrollList.swift

```diff
@@ -22,22 +25,25 @@ struct RefreshableInfiniteScrollList<Content: View>: View {
     ///
     /// - Parameters:
     ///   - isLoading: Whether the list is loading more content. Used to determine whether to show the infinite scroll indicator.
+    ///   - spacing: Spacing between each item in the list.
```

Comment:
Super nit: `spacing` is the first parameter, and in the comment, it is the second parameter.